ui/src/app/shared/header/header.component.html

```diff
@@ -67,9 +67,18 @@
     </button>
   </mat-form-field>
 
-  <div *ngIf="shouldDisplayStatusButtons()">
-    <button mat-raised-button class="status-button" (click)="showActiveJobs()">ACTIVE</button>
-    <button mat-raised-button class="status-button" (click)="showFailedJobs()">FAILED</button>
-    <button mat-raised-button class="status-button" (click)="showCompletedJobs()">COMPLETED</button>
+  <div class="search-table-controls">
+    <div *ngIf="shouldDisplayStatusButtons()">
+      <button mat-raised-button class="status-button" (click)="showActiveJobs()">ACTIVE</button>
+      <button mat-raised-button class="status-button" (click)="showFailedJobs()">FAILED</button>
+      <button mat-raised-button class="status-button" (click)="showCompletedJobs()">COMPLETED</button>
+    </div>
+
+    <mat-paginator *ngIf="jobs" #paginator
```

Comment:
Is it possible to get this paginator to right-align regardless of whether the status buttons are present or not? The current behavior is a little jarring.